src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -6479,6 +6487,14 @@ export interface Corporation extends WarehouseAPI, OfficeAPI {
    * @param percent - Percent of profit to issue as dividends.
    */
   issueDividends(percent: number): void;
+  /**
+   * Buyback Shares
+   */
+  buyBackShares(amt: number): void;
```

Comment:
Needs better param names (amount not amt), descriptions and @param annotations for the docs